### ISSUE TITLE
Set window icon

### ISSUE THREAD

**Issue by f-koehler** [f-koehler/KTailctl#57]
At first glance, organization and application name are set correctly in `main.cpp` and the name of the `.desktop` file is also correct. However, the window class still ends up being `org.kde.KTailctl` instead of `org.fkoehler.KTailctl` causing no application icon to be used 😢

**Comment by dtomvan** [comment]
Uhh, how can this even be? I can repro it but `rg -F 'org.kde.KTailctl'` just returns me nothing? And `<id>org.fkoehler.KTailctl</id>` is also set correctly...

**Comment by f-koehler** [comment]
> Uhh, how can this even be? I can repro it in flatpak but `rg -F 'org.kde.KTailctl'` just returns me nothing for both this repo and flathub/org.fkoehler.KTailctl? And `<id>org.fkoehler.KTailctl</id>` is also set correctly...

In Qt you set the organization and application name separately. For the organization the normal domain which is then reversed and the application gets appended (https://github.com/f-koehler/KTailctl/blob/eb16f7f99a30723f477d0c00efd4a10f58d75fe3/src/main.cpp#L39-L41). Your ripgrep command will thus not find the string you were looking for.  I thought I set this up correctly, I will try to look into any other KDE based apps to see how they do it.

**Comment by f-koehler** [comment]
Note to self: after looking at a non-KDE published APP, I might need to call `setDesktopFileName` (https://github.com/rinigus/pure-maps/blob/9e337b07d060296fa20ad97aee53fef57f42b688/src/main.cpp#L87)

**Comment by f-koehler** [comment]
Ok I found the issue. I will try to create a bug fix release today.

**Comment by dtomvan** [comment]
> In Qt you set the organization and application name separately. For the organization the normal domain which is then reversed and the application gets appended (
Yeah, I saw that, but no mentions of `kde.org` anyways. But thank you for the fix.
